### PR TITLE
env var bug fixes and doc updates

### PR DESCRIPTION
**Pull request by ritesh-modi**
Bug fixes 
 - added executing requirements.txt for local execution
 - renamed __init__.py file for web_classification
 - enforcing reading env vars as strings

### CODE REVIEW

**Review by prabdeb — APPROVED** [review]

**Review by mohanajuhi166 — COMMENTED** [review]
LGTM

File: docs/Azure_devops_how_to_setup.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ The `config.py` file located in the `llmops` folder contains the `EXECUTION_TYPE
 
 When the `EXECUTION_TYPE` variable is set to `"LOCAL"`, the templates will be executed on your local machine. This is useful for local development, testing, and debugging.
 
+Setup the environment by installing the required dependencies using the following command:
+
+``` bash
+pip install -r ./github/requirements/execute_job_requirements.txt
```
Comment (LittleColin) [inline docs/Azure_devops_how_to_setup.md:48]: This should be `.github`

File: docs/Azure_devops_how_to_setup.md [hunk 2]
```diff
@@ -62,6 +72,16 @@ There are two ways to execute templates on Azure:
 
 2. **Direct Execution from Local Machine**: You can also execute templates on Azure directly from your local machine. Set the `EXECUTION_TYPE` variable in `config.py` to `"AZURE"` and use the following command:
 
+Setup the environment by installing the required dependencies using the following command:
+
+``` bash
+pip install -r ./github/requirements/execute_job_requirements.txt
```
Comment (LittleColin) [inline docs/Azure_devops_how_to_setup.md:78]: Again `.github`

File: docs/github_workflows_how_to_setup.md [hunk 3]
```diff
@@ -46,6 +46,16 @@ When the `EXECUTION_TYPE` variable is set to `"LOCAL"`, the templates will be ex
 
 To execute a template locally, use the following command:
 
+Setup the environment by installing the required dependencies using the following command:
+
+``` bash
+pip install -r ./github/requirements/execute_job_requirements.txt
```
Comment (LittleColin) [inline docs/github_workflows_how_to_setup.md:52]: `.github`

File: docs/github_workflows_how_to_setup.md [hunk 4]
```diff
@@ -64,6 +74,16 @@ There are two ways to execute templates on Azure:
 
 2. **Direct Execution from Local Machine**: You can also execute templates on Azure directly from your local machine. Set the `EXECUTION_TYPE` variable in `config.py` to `"AZURE"` and use the following command:
 
+Setup the environment by installing the required dependencies using the following command:
+
+``` bash
+pip install -r ./github/requirements/execute_job_requirements.txt
```
Comment (LittleColin) [inline docs/github_workflows_how_to_setup.md:80]: `.github`

**Review by LittleColin — COMMENTED** [review]
Just `github` -> `.github`

**Pull request merged**